docs/concepts/authentication.md

```diff
@@ -94,7 +94,7 @@ With the ICM identity provider, for example, the user is then automatically logg
 
 ## Activate Identity Provider on specific paths
 
-For specific use cases the PWA have to activate the identity provider for specific pwa url paths.
+In future the PWA will have more cases which needs to activate the identity provider for specific pwa url paths.
```

Comment:
It is possible to activate the identity provider for specific pwa url paths, e.g.